.gitmodules

```diff
@@ -1,4 +1,3 @@
-[submodule "combirepo/rpmrebuild"]
```

Comment:
Why did the path changed? 
And how did it work before?

Reply:
Accidently added. Both paths work. I'll remove this change.

Reply:
Done